File: pythonlearning_2_15_25/myprograms/ex7languages/ex7.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("What is your favorite food? ");
        string favorite_food = Console.ReadLine();

        if (favorite_food.Equals("Rice Pudding", StringComparison.OrdinalIgnoreCase))
        {
            Console.WriteLine("Yes so nice!");
        }
        else
        {
            Console.WriteLine("Yuck!");
        }

        Console.WriteLine("Thanks for playing.");
    }
}

File: pythonlearning_2_15_25/myprograms/ex7languages/ex7.c
#include <stdio.h>
#include <string.h>

int main() {
    char favorite_food[50];

    printf("What is your favorite food? ");
    fgets(favorite_food, 50, stdin);
    favorite_food[strcspn(favorite_food, "\n")] = 0; // Remove newline character

    if (strcmp(favorite_food, "Rice Pudding") == 0) {
        printf("Yes so nice!\n");
    } else {
        printf("Yuck!\n");
    }

    printf("Thanks for playing.\n");
    return 0;
}
